src/taipy/core/config/data_node_config.py

```diff
@@ -383,6 +397,47 @@ def _configure_json(
         Config._register(section)
         return Config.sections[DataNodeConfig.name][id]
 
+    @staticmethod
+    def _configure_parquet(
+        id: str,
+        default_path: str = None,
+        exposed_type=_DEFAULT_EXPOSED_TYPE,
+        columns: Optional[List[str]] = None,
+        scope=_DEFAULT_SCOPE,
+        cacheable: bool = False,
```

Comment:
```suggestion
        cacheable: bool = False,
        compression: str = "snappy",
```
The compression property and its default value should be specifically defined.
This will require an update on "parquet.py" too.

Reply:
Since I think we don't have a preference on what compression algorithm to use, I deliberately omitted defining a default value here so as to use whatever pandas has set (`"snappy"` in this case). But I agree with your feedback, I'll write a test then to check that the pandas default is `"snappy"`, so that we can change the default if/when they do.

Reply:
I mean, on the documentation for example, we don't want to define:
> "The default value for `compression` is the same as the default value defined by `pandas`".

We should explicitly state what is the default value.

The same goes for the configure method and the ParquetDataNode.